demo/demodesk/config/settings.py

```diff
@@ -39,9 +33,10 @@
 # an internal demo you don't need such security, but please
 # remember when setting up your own development / production server!
 
+# Default teams mode to enabled unless overridden by an environment variable set to "false"
+HELPDESK_TEAMS_MODE_ENABLED=False if "false" == os.getenv("HELPDESK_TEAMS_MODE_ENABLED", "true").lower() else True
```

Comment:
I would invert the condition to make it more readable:

```suggestion
HELPDESK_TEAMS_MODE_ENABLED=False if os.getenv("HELPDESK_TEAMS_MODE_ENABLED", "true").lower() == "false" else True
```

Reply:
And why not even do it like this:

```suggestion
HELPDESK_TEAMS_MODE_ENABLED=os.getenv("HELPDESK_TEAMS_MODE_ENABLED", "true").lower() == "true"
```